opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -1562,7 +1562,7 @@ namespace Opm
         for (int perf = 0; perf < this->number_of_perforations_; ++perf) {
             const int cell_idx = this->well_cells_[perf];
             const auto& intQuants = simulator.model().intensiveQuantities(cell_idx, /*timeIdx=*/ 0);
-            const double trans_mult = simulator.problem().template rockCompTransMultiplier<double>(intQuants, cell_idx);
+            const double trans_mult = simulator.problem().template wellTransMultiplier<double>(intQuants, cell_idx);
```

Comment:
Here I include the perm reduction due to salt contribution to the output of the connection trans factor. The test failure are from this change. @plgbrts Is this inline with what you think should be outputted?